apps/deriv-hk/pages/help-centre.tsx

```diff
@@ -1,10 +1,11 @@
-import { Navigation } from '@deriv-com/features';
+import { Footer, Navigation } from '@deriv-com/features';
 import { HelpCenterTemplate } from '@deriv-com/templates';
 const HelpCenterPage = () => {
   return (
     <>
       <Navigation.Main />
       <HelpCenterTemplate />
+      <Footer.Main />
     </>
   );
 };
```

Comment:
@mohsen-deriv  are we planning to have another layer like `layout` that will have the `navigation` and `footer` so we don't need to add them on each pages?

Reply:
for now let's keep like this and when we're done identifying all the navigation variants, we can create the layout.